src/DynamoCoreWpf/Rendering/HelixRenderPackage.cs

```diff
@@ -79,6 +79,8 @@ public HelixRenderPackage()
         /// Set the transform that is applied to all geometry in the renderPackage.
         /// </summary>
         /// <param name="transform"></param>
+        // TODO: Unused method. Adds unnecessary dependency on ProtoGeometry. Remove in 3.0
+        [Obsolete]
```

Comment:
so these are not used internally, but they are quite useful for third parties as they do the hard work of converting the dynamo coordinate system into the helix coordinate system - for someone not familiar with this it's going to be hard to know to invert the axes and perform a rotation.

Reply:
Were they added for third parties in the first place?

Reply:
Usually we provide an alternative for obsolete methods, in this case, can you add obsoletion message maybe just indicating that we do not intend to public this in the first place?

Reply:
I did intend them to be public - and yes, I added them for myself acting as a third party dev writing a powerMill version of Dynamo where transforming geometry using helix was not possible and using protogeometry to do it was a non starter in terms of performance.

Reply:
I guess we can move these to protogeometry to avoid the tangled dependencies?

Reply:
or more simply, we could just create an interface for CoordinateSystems/matrices that represent the coordinate system for dynamo geometry.